src/main/java/build/buildfarm/instance/AbstractServerInstance.java

```diff
@@ -436,6 +444,14 @@ public String listOperations(
   }
 
   @Override
+  public void deleteOperation(String name) {
+    if (completedOperations.remove(name) == null &&
+        outstandingOperations.containsKey(name)) {
+      throw new UnsupportedOperationException();
```

Comment:
I'll go with IllegalStateException - I wanted to only allow 'unsupported' behavior to be passed through akin to the interface's only defined exceptional behavior, and then ignore the rest due to the Empty response.  This should probably be made into a concrete implementation requirement, rather than the abstract server.